### PR TITLE
Improve Ref Docs

### PR DESCRIPTION
**Pull request by sommersoft**
[Per Issue #566](https://github.com/adafruit/circuitpython/issues/566)

### CODE REVIEW

File: adafruit_ht16k33/segments.py [hunk 1]
```diff
@@ -20,13 +20,14 @@
...
-
-.. currentmodule:: adafruit_ht16k33.segments"""
+"""
+Segment Displays
+=================
+"""
 
 from adafruit_ht16k33.ht16k33 import HT16K33
 
+# pylint disable=bad-whitespace
```
Comment (kattni) [inline adafruit_ht16k33/segments.py]: I believe the white space issue here is that there should be two spaces before the `#`. If this is the case, I would rather we add the extra space than disable `pylint`. If that's not the issue, let me know and we can discuss the options.

**Review by kattni — CHANGES_REQUESTED** [review]
Thanks for doing this! Let me know what the white space issue specifically is so we can discuss it.

File: adafruit_ht16k33/segments.py (hunk 1 above)
Comment (sommersoft) [reply adafruit_ht16k33/segments.py]: Actually, it was a guess. Per the "my Travis is borked" commit, Travis was taking time in resyncing the deleted->re-forked repo. It showed me the overall status, but wouldn't load the log so I didn't know what was failing. I can easily remove the disable and see if it passes.

**Comment by sommersoft** [comment]
@kattni pylint disable removed; 5x5. 🌮 (started to type :thumsup:, :taco: appeared....made a judgement call)

**Review by kattni — APPROVED** [review]
Great job! Thanks for the update!

**Pull request merged**